src/header.jl

```diff
@@ -422,3 +422,33 @@ function show(io::IO, hdr::FITSHeader)
         i != n && println(io)
     end
 end
+
+"""
+    default_header(data::AbstractArray)
+
+Creates a default header for the given array with the `SIMPLE`, `BITPIX`, `NAXIS`, `NAXIS*`, and `EXTEND` entries.
+"""
+function default_header(data::AbstractArray{T}) where T <: Number
+    # assigning keys
+    hdu_keys = ["SIMPLE",
+                "BITPIX",
+                "NAXIS",
+                ("NAXIS$i" for i in 1:ndims(data))...,
+                "EXTEND"]
+
+    # assiging values
+    hdu_values = [true,                                           # SIMPLE
+                  Libcfitsio.bitpix_from_type(T),                 # BITPIX
```

Comment:
This module is deprecated now.  I guess this should be something like
```suggestion
                  CFITSIO.bitpix_from_type(T),                 # BITPIX
```